packages/azure-themes/src/azure/styles/DatePicker.styles.ts

```diff
@@ -11,33 +11,32 @@ export const DatePickerStyles = (props: IDatePickerStyleProps): Partial<IDatePic
   const { semanticColors } = theme;
   const TextHoverStyle = () => {
     return {
-      color: semanticColors.bodyText,
-      backgroundColor: StyleConstants.transparent,
+      color: 'black',
+      backgroundColor: '#f3f2f1',
+      //cursor: 'pointer'
     };
   };
   const TodayAndSelectedDayStyle = () => {
     return {
       '.ms-DatePicker-day-button.ms-DatePicker-day--today': {
-        backgroundColor: semanticColors.bodyBackground,
-        color: semanticColors.bodyText,
+        backgroundColor: '#0078D4', //semanticColors.bodyBackground,
```

Comment:
please remove commented code from all the changes you have made.
i'll stop repeating the same comment from here. :)

Reply:
Complete